### DISCUSSION TITLE
Laravel @can not working

### DISCUSSION THREAD

**Discussion by angeltecnology77** [spatie/laravel-permission#2126] [Q&A]
Hello, I installed laravel-permissions versions : * 5.5.4 in Laravel Framework version: 8.77.1, and I was able to create permissions, roles, assign permissions to roles and assign roles to users, but when I use the @can directive, nothing works, it doesn't recognize the permission.

It hides me, the link or the button for all users...for example...so

What can I do? Could you help me? please.

Thanks

**Comment by erikn69** [comment]
Duplicate of #1955, #1971, #2053, #1956
https://github.com/spatie/laravel-permission/blob/fc841f86190f558658c072c2f2107fdf67308c8f/tests/resources/views/can.blade.php#L1-L5
https://github.com/spatie/laravel-permission/blob/fc841f86190f558658c072c2f2107fdf67308c8f/tests/BladeTest.php#L76-L85
`@can` directive is tested and working, read documentation, you did forget something
